src/mesh/voxel_cloud.rs

```diff
@@ -128,111 +128,261 @@ impl VoxelCloud {
         voxel_cloud
     }
 
+    #[allow(dead_code)]
+    pub fn block_start(&self) -> &Point3<i32> {
+        &self.block_start
+    }
+
+    #[allow(dead_code)]
+    pub fn block_dimensions(&self) -> &Vector3<u32> {
+        &self.block_dimensions
+    }
+
+    #[allow(dead_code)]
+    pub fn voxel_dimensions(&self) -> &Vector3<f32> {
+        &self.voxel_dimensions
+    }
+
+    #[allow(dead_code)]
+    pub fn voxel_map_len(&self) -> usize {
+        self.voxel_map.len()
+    }
```

Comment:
I'm not sure if we want to replace the boolean array with anything else. Two reasons:

1. I've already relied too much on the existing data structure in the later features (sure, some refactor would do).
2. We'll most probably prefer using distance fields. In fact, I think we can eventually replace the current voxels entirely. I've developed the voxels further because it's a simplified version of the distance fields, so I thought it will be easier to implement and test the features with voxels and only then refactor it to distance fields. 

I'm not entirely sure how will it go, but I'll slowly start working on it and we'll see.

Btw, roughly, how much faster/efficient would the bit array be?

Regarding this specific method, I think it can be removed. I'm not using this outside of the struct anyway and form inside there are other, more explicit ways.

Reply:
> We'll most probably prefer using distance fields

Yes, bit arrays don't make sense for distance fields.

> Btw, roughly, how much faster/efficient would the bit array be?

Probably not much faster, but more space efficient.

> Regarding this specific method, I think it can be removed.

I am fine with that. The less dead code we have, the better.